src/Mod/TechDraw/App/DimensionFormatter.cpp

```diff
@@ -414,10 +414,12 @@ bool DimensionFormatter::isTooSmall(const double value, const QString& formatSpe
     QRegularExpression rxFormat(QStringLiteral("%[+-]?[0-9]*\\.*([0-9]*)[aefgrwAEFGRW]")); //printf double format spec
     QRegularExpressionMatch rxMatch = rxFormat.match(formatSpec);
     if (rxMatch.hasMatch()) {
-        QString decimalGroup = rxMatch.captured(1);
+        QString decimalGroup = rxMatch.captured(1);	// after the '.' but before the letter
         int factor = decimalGroup.toInt();
-        double minValue = pow(10.0, -factor);
-        if (std::fabs(value) < minValue) {
+        double minValue = pow(10.0, -factor);   //NOLINT
```

Comment:
Why we add `//NOLINT`? Magic number?

Reply:
Yes.  It seemed silly to make a constexpr for 10 just to use it for 10^x.